federation/pkg/federation-controller/util/cluster_util.go

```diff
@@ -146,3 +147,23 @@ func GetClientsetForCluster(cluster *federation_v1beta1.Cluster) (*fedclientset.
 	}
 	return nil, err
 }
+
+// Returns a bool to indicate if the object should be forwarded to a cluster
+func SendToCluster (cluster *federation_v1beta1.Cluster, objMeta metav1.ObjectMeta) bool {
+	forwardObject := true
+	//if there is an annotation for ClusterSelector
+	if val, ok := objMeta.Annotations["federation.beta.kubernetes.io/cluster-selector"]; ok {
+		//and it contains a valid ClusterSelector
```

Comment:
nit on comment structure: We generally prefer complete sentences as comments. The comments start with a capital letter and there is a space after `//`.